UI/Marker/GenericPlacemark.gd

```diff
@@ -2,16 +2,37 @@ extends Button
 class_name GenericPlacemark
 
 signal placemark_removed()
+var selectionpanel = null
 
+func ready():
+	connect("selection_panel", self, "receive_selection_panel")
```

Comment:
If the only purpose of this signal is to send from subclass to superclass the selection panel variable, why not just do ```selectionpanel = get_node("SelectionPanel")``` from the subclass ```_ready```? Or even just in here in the superclass, considering the child node is called the same in both enemy and item node cases